dotCMS/src/main/webapp/html/portlet/ext/sitesearch/site_search_audit.jsp

```diff
@@ -57,11 +56,19 @@
     </tr>
     <% for(SiteSearchAudit a : recents) { %>
     <%
-        StringBuilder hostList=new StringBuilder();
-        for(String hid : a.getHostList().split(",")) {
-            if(UtilMethods.isSet(hid))
-                hostList.append(APILocator.getHostAPI().find(hid, APILocator.getUserAPI().getSystemUser(), false).getHostname())
-                        .append("  ");
+        final StringBuilder hostList = new StringBuilder();
+        final String [] sites = a.getHostList().split(",");
```

Comment:
what is it happen if getHostList return null?

Reply:
shouldn't return null. The Audit data table demands some is written into that field at all times.